.github/workflows/maven-deploy.yml

```
@@ -17,6 +17,26 @@ jobs:
     - name: Build with Maven
       run: mvn -B package
 
+    - name: Generate sbom output
+      run: mvn org.cyclonedx:cyclonedx-maven-plugin:makeBom
+
+    - name: Install cycloneDX
+      run: |
+        wget https://github.com/CycloneDX/cyclonedx-cli/releases/download/v0.24.2/cyclonedx-linux-x64
+        chmod a+x cyclonedx-linux-x64
+      shell: bash
+
+    - name: Convert SBOM
+      run: |
+        ./cyclonedx-linux-x64 convert --input-format json --output-format spdxjson --input-file ./duo-client/target/cyclonedx-sbom.json --output-file spdx.json
+      shell: bash
```

What you have would work fine, but you could simplify it by reusing the conversion action from our other repo.
```suggestion
    - name: Convert SBOM
      uses: duosecurity/duo_client_python/.github/actions/sbom-convert@master
```